virttest/shared/cfg/base.cfg

```diff
@@ -1054,3 +1054,28 @@ uuid_dimm = ""
 # Note: For AMD SEV, you may also need the following settings:
 #       Disable rom loading for nic devices: nic_romfile = EMPTY_STRING
 #       Use OVMF.amdsev.fd if kernel-hashes is enabled: ovmf_code_filename = OVMF.amdsev.fd
+
+#
+# SGX guest params
```

Comment:
Since numa node is not  must required params for sgx, I suggest just adding the minimal params of enabling sgx:
use_mem = no
mem_devs += "epc0"
backend_mem_epc0 = "memory-backend-epc"
size_mem_epc0 = 64M
prealloc_epc0 = on
vm_sgx_epc_devs = sgx0
vm_sgx_epc_memdev_sgx0 = A
vm_sgx_epc_node_sgx0 = 0

Reply:
Better. We give an example to show to how configure it, the minimal configuration is simple and clear.

Reply:
update the cfg.